ztp/resource-generator/Containerfile

```diff
@@ -3,48 +3,40 @@ FROM openshift/origin-release:golang-1.16 as builder
 USER root
 ENV PKG_ROOT=cnf-features-deploy
 ENV PKG_PATH=/go/src/$PKG_ROOT
-ENV PGT_ROOT=$PKG_PATH/ztp/policygenerator
-ENV SC_ROOT=$PKG_PATH/ztp/siteconfig-generator
+ENV PGT_ROOT=$PKG_PATH/ztp/policygenerator-kustomize-plugin
+ENV SC_ROOT=$PKG_PATH/ztp/siteconfig-generator-kustomize-plugin
 RUN mkdir -p $PKG_PATH
 WORKDIR $PKG_PATH/
-ARG TAR_NAME
-COPY --chown=1001 $TAR_NAME  $PKG_PATH/
-RUN (tar -xf $TAR_NAME) && rm $TAR_NAME
-WORKDIR $PKG_PATH/$PKG_ROOT
-RUN go mod tidy && go mod vendor
+COPY . .
 WORKDIR $PGT_ROOT
-RUN go build -mod=vendor -o /PolicyGenTemplate
+RUN make build
 WORKDIR $SC_ROOT
-RUN go build -mod=vendor -o /SiteConfig
+RUN make build
 
 # Container image
 FROM ubi8-minimal
 USER root
-ENV WD=/home/ztp
-ENV TEMP=/tmp/ztp-temp
-ENV PGT_WD=/kustomize/plugin/ran.openshift.io/v1/policygentemplate
-ENV SITECONFIG_WD=/kustomize/plugin/ran.openshift.io/v1/siteconfig
+ENV BUILDER_ZTP=/go/src/cnf-features-deploy/ztp
+ENV ZTP_HOME=/home/ztp
+# Install utilities needed by the entrypoint scripts
 RUN microdnf install tar
```

Comment:
how much size install tar add to the container image ?

Reply:
Without tar:
```
REPOSITORY                              TAG         IMAGE ID      CREATED             SIZE
quay.io/redhat_emp1/ztp-site-generator  latest      c194ddc45eca  About a minute ago  116 MB
```

With tar:
```
REPOSITORY                              TAG         IMAGE ID      CREATED        SIZE
quay.io/redhat_emp1/ztp-site-generator  latest      816a5ee713f9  2 minutes ago  132 MB
```

So it adds ~16MB